int8 lift_command        # 1 = up, -1 = down
int8 gripper_command     # 1 = open, -1 = close
